### ISSUE TITLE
[Instance] Issue on Naxx 10 achievement

### ISSUE THREAD

**Issue by mmamani93** [dalaranwow/dalaran-wow#5376]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: We killed every boss in a single run, but on Sapphiron boss, the tank left raid, it seems that caused the boss not to count for us all.

**Expected behaviour**: The achieve should still have counted for us.

**Steps to reproduce the problem**:
1.  Attack a boss, leave raid. The boss will be your target and show as gray on selected target for the rest.
2. 
3. 

**Include proofs for this behaviour**
![image](https://user-images.githubusercontent.com/19556067/38203387-86988598-3675-11e8-8ed7-505e13eb9e3d.png)
![image](https://user-images.githubusercontent.com/19556067/38203438-9baa511e-3675-11e8-9fcb-ee0fcad51606.png)

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
going to add the achievement on JustDied manually

**Comment by TeridanDW** [comment]
Hello,
I don't think this is valid issue, because as @mmamani93 said, the boss was gray meaning tagged by someone outside of the group. It's the same for all creatures in the open world, if tagged by someone others don't get exp, quest credit, achievement, loot. That's how it works in wrath.

**Comment by Rushor** [comment]
Okay then /closed
